pkg/apiserver/user/module.go

```diff
@@ -11,24 +11,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package conprof
+package user
 
 import (
-	"github.com/pingcap/tidb-dashboard/pkg/apiserver/utils"
-	"github.com/pingcap/tidb-dashboard/pkg/config"
+	"github.com/pingcap/tidb-dashboard/pkg/utils"
 )
 
-var (
-	supportedTiDBVersions = []string{">= 5.3.0"}
-	featureSupported      *bool
-)
-
-func IsFeatureSupport(config *config.Config) (supported bool) {
-	if featureSupported != nil {
-		return *featureSupported
-	}
-
-	supported = utils.IsVersionSupport(config.FeatureVersion, supportedTiDBVersions)
-	featureSupported = &supported
-	return
-}
+var FeatureFlagNonRootLogin = utils.NewFeatureFlag("nonRootLogin", []string{">= 5.3.0"})
```

Comment:
Could you move this util to our new `util` directory? I think we'd better directly implement new utils in the `/util` instead of `/pkg/...`, in order to not being a block stone of transiting to the new `/util`.

Reply:
Maybe I can move this util after https://github.com/pingcap/tidb-dashboard/pull/1052 is merged.